pkg/restore/restore_test.go

```diff
@@ -2989,15 +2989,15 @@ func Test_resetVolumeBindingInfo(t *testing.T) {
 				WithName("pv-1").WithAnnotations(
 				KubeAnnBindCompleted,
 				KubeAnnBoundByController,
-				KubeAnnDynamicallyProvisioned,
```

Comment:
I don't know why need this change that removing the const variable and use string instead.